src/Utente.java

```diff
@@ -65,6 +74,18 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    public boolean login(String email, String password) {
+        return email.equals(getEmail()) && password.equals(getPassword());
+    }
 
+    public Roles getRole() {
+        return role;
+    }
 
+    public static List <Utente> leggiUtentiDaFile() throws FileNotFoundException {
```

Comment:
Potreste esternalizzare questo in una classe di utility, in modo da non dover aggiunere metodi statici alla classe utente

Reply:
una cosa simile al mapper